lsp-completion.el

```diff
@@ -184,7 +184,7 @@ This will help minimize popup flickering issue in `company-mode'."
 
 (defun lsp-completion--looking-back-trigger-characterp (trigger-characters)
   "Return character if text before point match any of the TRIGGER-CHARACTERS."
-  (unless (= (point) (point-at-bol))
+  (unless (= (point) (beginning-of-line))
```

Comment:
Just updated to get this change and now getting this backtrace when I try completing in my Go project:

```
Debugger entered--Lisp error: (wrong-type-argument number-or-marker-p nil)
  =(118 nil)
  (if (= (point) (beginning-of-line)) nil (seq-some #'(lambda (trigger-char) (and (equal (buffer-substring-no-properties (- ... ...) (point)) trigger-char) trigger-char)) trigger-characters))
  lsp-completion--looking-back-trigger-characterp(["."])
  lsp-completion-at-point()
  completion--capf-wrapper(lsp-completion-at-point optimist)
  company--capf-wrapper(lsp-completion-at-point optimist)
  company--capf-data-real()
  company--capf-data()
  company-capf(prefix)
  apply(company-capf prefix)
  company-call-backend-raw(prefix)
  company--force-sync(company-call-backend-raw (prefix) company-capf)
  company-call-backend(prefix)
  company--begin-new()
  company--perform()
  company-auto-begin()
  company-idle-begin(#<buffer main.go> #<window 9 on main.go> 102 118)
  apply(company-idle-begin (#<buffer main.go> #<window 9 on main.go> 102 118))
  timer-event-handler([t 25502 835 435308 nil company-idle-begin (#<buffer main.go> #<window 9 on main.go> 102 118) nil 0 nil])
```

If I change back to `point-at-bol` it works. (The top of the backtrace here is maybe funky because I eval'd to test changing back to `point-at-bol`, then eval'd again using `beginning-of-line` to get the backtrace to paste here)

Reply:
Ops, it should be `line-beginning-position`.

Reply:
👍 Tried that and it does work.